src/native/macos.rs

```diff
@@ -901,6 +901,40 @@ unsafe fn set_icon(ns_app: ObjcId, icon: &Icon) {
     CGImageRelease(image);
 }
 
+/// Initialize the system menu bar for this application
+/// - ns_app: This NSApplication
+#[no_mangle]
+unsafe fn initialize_menu_bar(ns_app: ObjcId) {
+    // Adapted from Winit `menu::initialize`
+
+    // System menu bar
+    let menu_bar: ObjcId = msg_send![class!(NSMenu), new];
+    // Entry for the app menu dropdown in the menu bar
+    let app_menu_item: ObjcId = msg_send![class!(NSMenuItem), new];
+    let app_menu: ObjcId = msg_send![class!(NSMenu), new];
+
+    // Hook up the menu components to the application
+    let () = msg_send![app_menu_item, setSubmenu: app_menu];
```

Comment:
up to you, but there is a special macro with a dash, `msg_send_![app_menu_item, setSubmenu: app_menu];`

Reply:
Oh that's nicer looking. Switched to using that.